tests/jax/common.libsonnet

```diff
@@ -80,56 +89,6 @@ local tpus = import 'templates/tpus.libsonnet';
 
   JaxPodTest:: self.JaxTest {
```

Comment:
Since `JaxPodTest` is a copy of `JaxTest`, you can remove `JaxPodTest` and replace it with `JaxTest` wherever it's used

I believe you can do the same for `PodFlaxLatest` (feel free to do this in a follow-up if it's not trivial): 

https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/8d5d7e9dcfe96b4b4d011ef0d2e5eafeafc0cff3/tests/jax/latest/common.libsonnet#L64

Reply:
yeah it is true it seems.